src/Symfony/Component/Form/Extension/Core/Type/FormFlowType.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\Core\Type;
+
+use Symfony\Component\Form\Flow\AbstractFlowType;
+use Symfony\Component\Form\Flow\DataAccessor\PropertyPathStepAccessor;
+use Symfony\Component\Form\Flow\DataAccessor\StepAccessorInterface;
+use Symfony\Component\Form\Flow\DataStorage\DataStorageInterface;
+use Symfony\Component\Form\Flow\DataStorage\NullDataStorage;
+use Symfony\Component\Form\Flow\FormFlowBuilderInterface;
+use Symfony\Component\Form\Flow\FormFlowInterface;
+use Symfony\Component\Form\Flow\FormFlowStepView;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\OptionsResolver\Exception\MissingOptionsException;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\PropertyAccess\PropertyAccess;
+use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
+use Symfony\Component\PropertyAccess\PropertyPath;
+use Symfony\Component\PropertyAccess\PropertyPathInterface;
+
+/**
+ * A multistep form.
```

Comment:
a question about naming:

you describe this as a form multistep 
but the class is named form flow

its feels weird at first read

Reply:
They mean the same to me... I’m just phrasing it differently to make the idea clearer. What is the weird part?

Reply:
"flow" vs "multistep" was what I meant
multistep is clear to me, I do also prefer this naming over for class name type